test/mitmproxy/tools/console/test_statusbar.py

```diff
@@ -13,14 +13,12 @@ def test_statusbar(monkeypatch):
         stickycookie="~dst example.com",
         stickyauth="~dst example.com",
         default_contentview="javascript",
-        console_order="url",
```

Comment:
Let's add the addon here, because these actually should be set to have better coverage of the statusbar :)